template/src/routes/router.tsx

```diff
@@ -17,7 +15,7 @@ const renderRoutes = (routeData: RouteType[]) => (
       key={data.path}
       exact={data.exact}
       path={data.path}
-      component={data.component}
+      component={RouteComponent.get(data.name)}
```

Comment:
If you follow my previous comment then this should be:

```suggestion
      component={RouteComponent[data.name]}
```